stable/weave-scope/charts/weave-scope-frontend/templates/deployment.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    {{- include "helm_std_labels" . | indent 4 }}
+    component: frontend
+  name: {{ template "fullname" . }}-app
+  annotations:
+    {{- include "scope_annotations" . | indent 4 }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ template "name" . }}
+      release: {{ .Release.Name }}
+      component: frontend
+  template:
+    metadata:
+      labels:
+        {{- include "helm_std_labels" . | indent 8 }}
+        component: frontend
+    spec:
+      containers:
+        - name: app
+          image: {{ .Values.global.image | default "weaveworks/scope" }}:{{ .Values.global.imageTag | default "1.6.2" }}
+          imagePullPolicy: {{ .Values.imagePullPolicy | default "" | quote }}
+          args:
+            - '--no-probe'
+          ports:
+            - containerPort: {{ .Values.containerPort | default 4040 }}
+              protocol: TCP
+          {{- include "scope_resources" . | indent 10 }}
```

Comment:
There's no need for a template function. That would just make it overly complicated. This is how it is commonly done:

```yaml
resources:
{{ toYaml .Values.resources | indent 12 }}
```